### ISSUE TITLE
Registration Problem --  Error Occurred Message  popup followed by 404 error

### ISSUE THREAD

**Issue by uskudarli** [bounswe/bounswe2015group1#174]
I tried to login with my earlier registration. suzan.uskudarli@boun.edu.tr but it failed. So I tried to register again, and got the following error:

![mamaregistration](https://cloud.githubusercontent.com/assets/1347035/12146310/ba9648d4-b49b-11e5-8d9b-de752d93f341.png)

I had given a ridiculous birth date in previous registration as well. It did accept it. So I don't think it is related to that.

In any case, the message is not okay. Totally not explanatory and user can't move forward. 

Then, it gets even worse and I get redirected to page that says:

HTTP ERROR 404

Problem accessing /register. Reason:

    Not Found

Powered by Jetty://

**Comment by sleepnir** [comment]
This is probably caused by trying to register with the same email. In case of errors, server can return an error object which includes error reason. So I am assigning this to backend team. @onurmus @grkmndr

**Comment by uskudarli** [comment]
Any idea, why i could not login with the previous registration? I mean, why did it not work? I though perhaps you reset the database. 

Obviously the user is going to need some good feedback, rather than to seeing something went wrong. Error handing 101 :)

Hopefully, it is a simple fix. Thanks.

**Comment by grkmndr** [comment]
Yes we indeed reset the database a couple of days ago that's why you could not login with your previous credentials. We are working on the birthdate and register problem right now and it seems like a simple fix.
